SmartSaver.MVC/Controllers/TransactionsController.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using SmartSaver.EntityFrameworkCore;
+using SmartSaver.EntityFrameworkCore.Models;
+using SmartSaver.MVC.Models;
+
+namespace SmartSaver.MVC.Controllers
+{
+    public class TransactionsController : Controller
+    {
+        private readonly ApplicationDbContext _context;
+        private readonly ILogger<TransactionsController> _logger;
+
+        public TransactionsController(ApplicationDbContext context, ILogger<TransactionsController> logger)
+        {
+            _context = context;
+            _logger = logger;
+        }
+
+        // GET: TransactionsController
```

Comment:
cannot understand what are these comments for. Need more clarification, maybe use /// instead of //

Reply:
This is default generated. Please, read about REST.

Reply:
Seems like I lack of knowledge to understand the code at this point :dizzy_face: